BossMod/BossModule/BossModuleConfig.cs

```diff
@@ -60,5 +61,11 @@ public class BossModuleConfig : ConfigNode
 
         [PropertyDisplay("Always show all alive party members")]
         public bool ShowIrrelevantPlayers = false;
+        [PropertyDisplay("In world curve accuracy (lower is better, but processing time goes up)")]
+        [PropertySlider(0.001f, 1, Speed = 0.001f)]
+        public float WorldCurveError = 1;
+        [PropertyDisplay("Radar curve accuracy (lower is better, but processing time goes up)")]
+        [PropertySlider(0.001f, 0.3f, Speed = 0.001f)]
+        public float RadarCurveError = 0.3f;
```

Comment:
Yes, on my machine the performance cost is relatively small, but some people play with ancient machines so who knows what it will do there. But I understand if you will just delete the pull request. I think 0.1f would be a good number for both, for less you really need a magnifying glass to notice.